src/main/resources/assets/dragonsurvival/lang/en_us.json

```diff
@@ -1387,11 +1387,11 @@
   "ds.advancements.fire_bolas.title": "Trapper",
   "ds.advancements.fire_bolas.description": "Fire a crossbow with the bolas enchantment. Time to catch dragons!",
   "ds.advancements.be_young_dragon.title": "Growth Process",
-  "ds.advancements.be_young_dragon.description": "As time goes on, the dragon grows",
+  "ds.advancements.be_young_dragon.description": "As time goes on, the dragon grows...",
   "ds.advancements.be_adult_dragon.title": "Older And Stronger",
-  "ds.advancements.be_adult_dragon.description": "Eating the hearts of the elder dragons aids in growth.",
+  "ds.advancements.be_adult_dragon.description": "Each day, you get stronger...",
   "ds.advancements.be_full_growth_dragon.title": "Just Older",
-  "ds.advancements.be_full_growth_dragon.description": "It seems to have become harder to fit through the regular doors now.",
+  "ds.advancements.be_full_growth_dragon.description": "Now you are old and strong, strong, strong!",
```

Comment:
Remove the information that there are large doors and replace it with...? Is that a reference to something instead of information?